lib/toPaths.js

```diff
@@ -1,13 +1,18 @@
 var isArray = Array.isArray;
 var typeOfObject = "object";
+var typeOfString = "string";
+var typeOfNumber = "number";
+var MAX_SAFE_INTEGER = 9007199254740991; // Number.MAX_SAFE_INTEGER in es6
```

Comment:
I figured as the number is fixed we may as well use the shorter code here. (Alternative `var MAX_SAFE_INTEGER = Number.MAX_SAFE_INTEGER || 9007199254740991;`)